static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -152,33 +153,32 @@ const addMobileInlineAds = () => {
 				minAbove: 35,
 				minBelow: 200,
 			},
-			fromBottom: true,
 		},
+		// TODO this looks like it was mistakenly put in the selectors object.
+		// What is the impact of correcting the typo??
+		fromBottom: true,
```

Comment:
Great catch! I guess if it was placed in the wrong object it was effectively being ignored, and now we're going to try spacefilling from bottom to top? Do we think that'll have a commercial impact at all?

Reply:
Yes I think that's right. And it seems like the sort of change that would have an unpredictable impact on location and number of slots. 
Maybe the best thing to do for now is to leave the property where it is with a comment and then investigate later with the help of @simonbyford 's handy debugger and maybe an experiment.